src/app/personside/visittkort-v2/body/familie/Familie.tsx

```diff
@@ -6,17 +6,20 @@ import Sivilstand from './Sivilstand';
 import ListeAvBarn from './ListeAvBarn';
 
 interface Props {
+    feilendeSystem: boolean;
```

Comment:
[NITPICK] navngiving av variablen gjør at det for meg virker som om typen burde vært `InformasjonElement` siden andre steder i koden har vi `feilendeSystemer: Array<InformasjonElement>`.

Ofte når det er `boolean` verdier vi sender så bruker man `erXXXXXX` eller `harXXXXXXX`, så det er kanskje en mulighet her, evt kan man bruke noe alà `harFeil`, `manglerData` eller tilsvarende